### DISCUSSION TITLE
H1-6.0-e-g2 with WIFI Stick

### DISCUSSION THREAD

**Discussion by vergilet** [nathanmarlor/foxess_modbus#690] [Q&A] [answered]
Is there any guide / tutorial how to setup local connection with WIFI stick?
https://www.fox-ess.com/download/upfiles/10-203-00011-06%20WiFi2.0%20English%20Installation%20guide.pdf
[SmartWifiQuickGuide2_0 (1) (1).pdf](https://github.com/user-attachments/files/17268659/SmartWifiQuickGuide2_0.1.1.pdf)

There is also UI not like in pdfs:

<img width="1183" alt="Знімок екрана 2024-10-06 о 11 36 46" src="https://github.com/user-attachments/assets/ada09810-4645-45b0-a001-09e6fe1f155f">
<img width="1308" alt="Знімок екрана 2024-10-06 о 11 37 33" src="https://github.com/user-attachments/assets/3c9342f4-1129-4d17-9696-aa6b874601df">
<img width="1310" alt="Знімок екрана 2024-10-06 о 11 37 47" src="https://github.com/user-attachments/assets/429e9336-fbb3-44d4-bc94-e3293d969cee">
<img width="1297" alt="Знімок екрана 2024-10-06 о 11 37 58" src="https://github.com/user-attachments/assets/91f4aa8a-34d9-4ecd-ba49-0ad972c53f85">

Also want to mention that port selection are all undefinied :( 

Any ideas?

**Comment by canton7** [comment, marked as answer]
This integration cannot use the wifi stick, and must be connected to the modbus connectors on the inverter I'm afraid.
